Stack/Opc.Ua.Core/Stack/Bindings/BufferManager.cs

```diff
@@ -117,14 +118,16 @@ public class BufferManager
         /// Constructs the buffer manager.
         /// </summary>
         /// <param name="name">The name.</param>
-        /// <param name="maxPoolSize">Max size of the pool.</param>
+        /// <param name="maxBufferCount">Max allowed number of buffers to be used</param>
         /// <param name="maxBufferSize">Max size of the buffer.</param>
-        public BufferManager(string name, int maxPoolSize, int maxBufferSize)
+        public BufferManager(string name, int maxBufferCount, int maxBufferSize)
```

Comment:
maxBufferCount and m_maxBufferCount can be removed

Reply:
removed unused variables